docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1919,6 +1919,47 @@ The series containing such samples are skipped during ingestion, and valid serie
 When `-ingester.error-sample-rate` is configured to a value greater than `0`, invalid native histogram errors are logged only once every `-ingester.error-sample-rate` times.
 {{< /admonition >}}
 
+### err-mimir-native-histogram-custom-buckets-mismatch
+
+This non-critical error occurs when Mimir receives a write request that contains a sample that is a native histogram
+with custom buckets where more buckets are present than the upper bound is specified for. In technical terms, the
+number of bucket values from spans is higher than the number of custom values.
+
+{{< admonition type="note" >}}
+Series containing these samples are skipped during ingestion. Valid series in the same request are ingested.
+{{< /admonition >}}
+
+{{< admonition type="note" >}}
+When you configure `-ingester.error-sample-rate` to a value greater than `0`, invalid native histogram errors are logged only once every `-ingester.error-sample-rate` times.
```

Comment:
@tacole02 how about:
```suggestion
When you configure `-ingester.error-sample-rate` to a value of `N` that is greater than `0`, only every `Nth` invalid native histogram error is logged.
```

Reply:
Note: these two notes are copy pasted from existing docs they occur dozens of times.

Reply:
I think that sounds clear and more specific than the original! Thanks, @krajorama !